pkg/manager/member/tikv_member_manager.go

```diff
@@ -511,6 +511,37 @@ func volumeClaimTemplate(r corev1.ResourceRequirements, metaName string, storage
 	}
 }
 
+func transformTiKVConfigMap(src []byte, tc *v1alpha1.TidbCluster) string {
+	config := tc.Spec.TiKV.Config
+	if config == nil {
+		return string(src)
+	}
+	srcStr := string(src)
+	if config.TiKVPessimisticTxn != nil {
+		clusterVersionGE4, err := clusterVersionGreaterThanOrEqualTo4(tc.TiKVVersion())
+		if err != nil {
+			klog.Infof("cluster version: %s is not semantic versioning compatible", tc.TiKVVersion())
```

Comment:
we assume the version is greater or equal to 4 on error, so return the configuration here and we don't need to check error later